internal/jobservice/eventstojobs/eventstojobs.go

```diff
@@ -105,7 +105,12 @@ func (eventToJobService *EventsToJobService) streamCommon(ctx context.Context, t
 				jobStatus := EventsToJobResponse(*msg.Message)
 				if jobStatus != nil {
 					jobStatus := repository.NewJobStatus(eventToJobService.queue, eventToJobService.jobSetId, currentJobId, *jobStatus)
-					eventToJobService.jobServiceRepository.UpdateJobServiceDb(jobStatus)
+					err := eventToJobService.jobServiceRepository.UpdateJobServiceDb(jobStatus)
+					if err != nil {
+						log.WithError(err).Error("could not update job status, retrying")
```

Comment:
Yes I think it could conceivably cause an infinite loop. I modeled this on the error handling just above at https://github.com/G-Research/armada/pull/1568/files/262bad849730c45e0575070af442f6bf1a84d1f1#diff-512415eae5bb0e754a65e8a3cdd55a69b04cd1a13a6e62b806b5fa8a33dd5d61L99

Reply:
I don't mind updating these to retry a few times and then either panic or just continue on.